test.js

```diff
@@ -172,3 +171,17 @@ test('should deserialize plain object', t => {
 	t.is(deserialized.name, 'name');
 	t.is(deserialized.code, 'code');
 });
+
+test('common properties should not be enumerable', t => {
+	const object = {
+		message: 'error message',
+		stack: 'at <anonymous>:1:13',
+		name: 'name',
+		code: 'code'
```

Comment:
`code` is enumerable, at least in Node.js:

```js
> try {
... 	fs.readFileSync('sadasd')
... } catch (err) {
... 	console.log(Object.keys(err));
... }
[ 'errno', 'syscall', 'code', 'path' ]
```

Reply:
Ah, good catch!